tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -115,7 +116,10 @@ public static void Initialize()
                         enabledCategories |= InstrumentationCategory.AppSec;
                     }
 
-                    var defs = NativeMethods.RegisterCallTargetDefinitions("Tracing", InstrumentationDefinitions.Instrumentations, (uint)enabledCategories);
+                    var defs = _useEmbeddedDefinitions ?
+                                    NativeMethods.InitEmbeddedCallTargetDefinitions(ConfigTelemetryData.ManagedTracerTfmValue, (uint)enabledCategories) :
```

Comment:
nit: we should probably move `ConfigTelemetryData.ManagedTracerTfmValue` to somewhere else, seeing as it's not just a telemetry variable now, e.g. `TracerConstants`